### DISCUSSION TITLE
After following setup, getting 403 error on basic test in editor

### DISCUSSION THREAD

**Discussion by ConnKing** [Elringus/UnityGoogleDrive#113] [Q&A]
I ran through the setup guide and tried the .json OAuth 2.0 Client ID files for both Desktop and Web application with http://localhost listed in under my Authorized redirect URIs.
The values seem to be getting set correctly in the window
<img width="658" alt="image" src="https://user-images.githubusercontent.com/110412626/228373876-a8174e83-cf92-4fae-80b0-cb48ecdb108c.png">

Then on start I call:
```
 void Start()
    {
        // Listing files.
        GoogleDriveFiles.List().Send().OnDone += fileList => {
            var files = fileList.Files;
            foreach (var f in files)
            {
                Debug.Log(f);

            }
        };
}

```
When I click run in the editor with the script attached to a GameObject in the scene. It gives me this error:
**UnityGoogleDrive: HTTP/1.1 403 Forbidden**
**Google Drive API Error Description: Code '403' Message: 'The request is missing a valid API key.'**
 **- Domain: 'global' Reason: 'forbidden' Message: 'The request is missing a valid API key.' LocationType: '' Location: ''**
 
 the uri listed with the error is https://www.googleapis.com/drive/v3/files

Is there something I'm missing? Any help would be appreciated.

**Comment by narino** [comment]
up. same problem :(

**Comment by elringus** [comment]
Here is the associated issue: #114
